src/main/jbake/templates/footer.ftl

```diff
@@ -40,5 +40,24 @@
     	  link.parents('li,ul').addClass('active');
 	
 	</script>
+
+    <!-- Matomo -->
+    <script>
+        var _paq = window._paq = window._paq || [];
+        /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+        /* We explicitly disable cookie tracking to avoid privacy issues */
+        _paq.push(['disableCookies']);
+        /* Measure a visit to flink.apache.org and nightlies.apache.org/flink as the same visit */
```

Comment:
I think the comment above was copy-pasted, but it's actually opennlp.apache.org (there's no opennlp under nightlies).

Reply:
Yes, this is true. It is copy pasted from the Privacy Slack, there Martjin Visser posted the snippet for us. I will change the flink comment ;)

Reply:
Thanks! I think this is the first ASF I participate that I know Matomo is used. Looking forward to learning how it works and maybe suggest to other projects I contribute to to try that too.

Reply:
Maven is using it for quite a while now and the list is growing but it isn't promoted across the TLPs.